Run #1: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v3
Run #1: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v4
Run #1: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v5
Traceback (most recent call last):
  File "/cluster/home/shmittal/LLMIdentifiers/main.py", line 211, in <module>
    answer_0 = extract_json(agent_0.staghunt_prompt(version=game_prompt_version))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 107, in staghunt_prompt
    return self.gen(resp_format="json_object")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 54, in gen
    raise ValueError(f"ERROR: {response}")
ValueError: ERROR: ChatCompletion(id=None, choices=None, created=None, model=None, object=None, service_tier=None, system_fingerprint=None, usage=None, error={'message': 'Provider returned error', 'code': 400, 'metadata': {'raw': '{\n  "error": {\n    "message": "\'messages\' must contain the word \'json\' in some form, to use \'response_format\' of type \'json_object\'.",\n    "type": "invalid_request_error",\n    "param": "messages",\n    "code": null\n  }\n}', 'provider_name': 'Azure'}}, user_id='user_2sYL3EYaIPaK4wHQX6V27nHTfMt')
Run #2: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v3
Run #2: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v4
Run #2: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v5
Traceback (most recent call last):
  File "/cluster/home/shmittal/LLMIdentifiers/main.py", line 211, in <module>
    answer_0 = extract_json(agent_0.staghunt_prompt(version=game_prompt_version))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 107, in staghunt_prompt
    return self.gen(resp_format="json_object")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 54, in gen
    raise ValueError(f"ERROR: {response}")
ValueError: ERROR: ChatCompletion(id=None, choices=None, created=None, model=None, object=None, service_tier=None, system_fingerprint=None, usage=None, error={'message': 'Provider returned error', 'code': 400, 'metadata': {'raw': '{\n  "error": {\n    "message": "\'messages\' must contain the word \'json\' in some form, to use \'response_format\' of type \'json_object\'.",\n    "type": "invalid_request_error",\n    "param": "messages",\n    "code": null\n  }\n}', 'provider_name': 'Azure'}}, user_id='user_2sYL3EYaIPaK4wHQX6V27nHTfMt')
Run #3: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v3
Run #3: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v4
Run #3: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v5
Traceback (most recent call last):
  File "/cluster/home/shmittal/LLMIdentifiers/main.py", line 211, in <module>
    answer_0 = extract_json(agent_0.staghunt_prompt(version=game_prompt_version))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 107, in staghunt_prompt
    return self.gen(resp_format="json_object")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 54, in gen
    raise ValueError(f"ERROR: {response}")
ValueError: ERROR: ChatCompletion(id=None, choices=None, created=None, model=None, object=None, service_tier=None, system_fingerprint=None, usage=None, error={'message': 'Provider returned error', 'code': 400, 'metadata': {'raw': '{\n  "error": {\n    "message": "\'messages\' must contain the word \'json\' in some form, to use \'response_format\' of type \'json_object\'.",\n    "type": "invalid_request_error",\n    "param": "messages",\n    "code": null\n  }\n}', 'provider_name': 'Azure'}}, user_id='user_2sYL3EYaIPaK4wHQX6V27nHTfMt')
Run #4: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v3
Run #4: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v4
Run #4: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v5
Traceback (most recent call last):
  File "/cluster/home/shmittal/LLMIdentifiers/main.py", line 211, in <module>
    answer_0 = extract_json(agent_0.staghunt_prompt(version=game_prompt_version))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 107, in staghunt_prompt
    return self.gen(resp_format="json_object")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 54, in gen
    raise ValueError(f"ERROR: {response}")
ValueError: ERROR: ChatCompletion(id=None, choices=None, created=None, model=None, object=None, service_tier=None, system_fingerprint=None, usage=None, error={'message': 'Provider returned error', 'code': 400, 'metadata': {'raw': '{\n  "error": {\n    "message": "\'messages\' must contain the word \'json\' in some form, to use \'response_format\' of type \'json_object\'.",\n    "type": "invalid_request_error",\n    "param": "messages",\n    "code": null\n  }\n}', 'provider_name': 'Azure'}}, user_id='user_2sYL3EYaIPaK4wHQX6V27nHTfMt')
Run #5: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v3
Run #5: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v4
Run #5: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v5
Traceback (most recent call last):
  File "/cluster/home/shmittal/LLMIdentifiers/main.py", line 211, in <module>
    answer_0 = extract_json(agent_0.staghunt_prompt(version=game_prompt_version))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 107, in staghunt_prompt
    return self.gen(resp_format="json_object")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 54, in gen
    raise ValueError(f"ERROR: {response}")
ValueError: ERROR: ChatCompletion(id=None, choices=None, created=None, model=None, object=None, service_tier=None, system_fingerprint=None, usage=None, error={'message': 'Provider returned error', 'code': 400, 'metadata': {'raw': '{\n  "error": {\n    "message": "\'messages\' must contain the word \'json\' in some form, to use \'response_format\' of type \'json_object\'.",\n    "type": "invalid_request_error",\n    "param": "messages",\n    "code": null\n  }\n}', 'provider_name': 'Azure'}}, user_id='user_2sYL3EYaIPaK4wHQX6V27nHTfMt')
Run #6: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v3
Run #6: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v4
Run #6: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v5
Traceback (most recent call last):
  File "/cluster/home/shmittal/LLMIdentifiers/main.py", line 211, in <module>
    answer_0 = extract_json(agent_0.staghunt_prompt(version=game_prompt_version))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 107, in staghunt_prompt
    return self.gen(resp_format="json_object")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 54, in gen
    raise ValueError(f"ERROR: {response}")
ValueError: ERROR: ChatCompletion(id=None, choices=None, created=None, model=None, object=None, service_tier=None, system_fingerprint=None, usage=None, error={'message': 'Provider returned error', 'code': 400, 'metadata': {'raw': '{\n  "error": {\n    "message": "\'messages\' must contain the word \'json\' in some form, to use \'response_format\' of type \'json_object\'.",\n    "type": "invalid_request_error",\n    "param": "messages",\n    "code": null\n  }\n}', 'provider_name': 'Azure'}}, user_id='user_2sYL3EYaIPaK4wHQX6V27nHTfMt')
Run #7: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v3
Run #7: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v4
Run #7: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v5
Traceback (most recent call last):
  File "/cluster/home/shmittal/LLMIdentifiers/main.py", line 211, in <module>
    answer_0 = extract_json(agent_0.staghunt_prompt(version=game_prompt_version))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 107, in staghunt_prompt
    return self.gen(resp_format="json_object")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 54, in gen
    raise ValueError(f"ERROR: {response}")
ValueError: ERROR: ChatCompletion(id=None, choices=None, created=None, model=None, object=None, service_tier=None, system_fingerprint=None, usage=None, error={'message': 'Provider returned error', 'code': 400, 'metadata': {'raw': '{\n  "error": {\n    "message": "\'messages\' must contain the word \'json\' in some form, to use \'response_format\' of type \'json_object\'.",\n    "type": "invalid_request_error",\n    "param": "messages",\n    "code": null\n  }\n}', 'provider_name': 'Azure'}}, user_id='user_2sYL3EYaIPaK4wHQX6V27nHTfMt')
Run #8: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v3
Run #8: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v4
Run #8: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v5
Traceback (most recent call last):
  File "/cluster/home/shmittal/LLMIdentifiers/main.py", line 211, in <module>
    answer_0 = extract_json(agent_0.staghunt_prompt(version=game_prompt_version))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 107, in staghunt_prompt
    return self.gen(resp_format="json_object")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 54, in gen
    raise ValueError(f"ERROR: {response}")
ValueError: ERROR: ChatCompletion(id=None, choices=None, created=None, model=None, object=None, service_tier=None, system_fingerprint=None, usage=None, error={'message': 'Provider returned error', 'code': 400, 'metadata': {'raw': '{\n  "error": {\n    "message": "\'messages\' must contain the word \'json\' in some form, to use \'response_format\' of type \'json_object\'.",\n    "type": "invalid_request_error",\n    "param": "messages",\n    "code": null\n  }\n}', 'provider_name': 'Azure'}}, user_id='user_2sYL3EYaIPaK4wHQX6V27nHTfMt')
Run #9: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v3
Run #9: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v4
Run #9: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v5
Traceback (most recent call last):
  File "/cluster/home/shmittal/LLMIdentifiers/main.py", line 211, in <module>
    answer_0 = extract_json(agent_0.staghunt_prompt(version=game_prompt_version))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 107, in staghunt_prompt
    return self.gen(resp_format="json_object")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 54, in gen
    raise ValueError(f"ERROR: {response}")
ValueError: ERROR: ChatCompletion(id=None, choices=None, created=None, model=None, object=None, service_tier=None, system_fingerprint=None, usage=None, error={'message': 'Provider returned error', 'code': 400, 'metadata': {'raw': '{\n  "error": {\n    "message": "\'messages\' must contain the word \'json\' in some form, to use \'response_format\' of type \'json_object\'.",\n    "type": "invalid_request_error",\n    "param": "messages",\n    "code": null\n  }\n}', 'provider_name': 'Azure'}}, user_id='user_2sYL3EYaIPaK4wHQX6V27nHTfMt')
Run #10: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v3
Run #10: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v4
Run #10: gpt vs deepseek -> Game: staghunt with SPV: v1 and GPV: v5
Traceback (most recent call last):
  File "/cluster/home/shmittal/LLMIdentifiers/main.py", line 211, in <module>
    answer_0 = extract_json(agent_0.staghunt_prompt(version=game_prompt_version))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 107, in staghunt_prompt
    return self.gen(resp_format="json_object")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/shmittal/LLMIdentifiers/prompt_engine.py", line 54, in gen
    raise ValueError(f"ERROR: {response}")
ValueError: ERROR: ChatCompletion(id=None, choices=None, created=None, model=None, object=None, service_tier=None, system_fingerprint=None, usage=None, error={'message': 'Provider returned error', 'code': 400, 'metadata': {'raw': '{\n  "error": {\n    "message": "\'messages\' must contain the word \'json\' in some form, to use \'response_format\' of type \'json_object\'.",\n    "type": "invalid_request_error",\n    "param": "messages",\n    "code": null\n  }\n}', 'provider_name': 'Azure'}}, user_id='user_2sYL3EYaIPaK4wHQX6V27nHTfMt')
